.github/ISSUE_TEMPLATE/request-new-package.yml

```diff
@@ -11,6 +11,12 @@ body:
           placeholder: https://www.example.com
       validations:
           required: true
+  - type: checkboxes
```

Comment:
The indentation of your inserted lines do not match the others. As a result, the template is invalid. Take a look at https://github.com/getsolus/packages/blob/a782116016fe1e5c38fa60353e0c705fc0fbdc9b/.github/ISSUE_TEMPLATE/request-new-package.yml and you'll see an error.

Once the indentation is fixed, this is good to go.

Reply:
Should now be fixed.